server/v2/tests/auth.spec.js

```diff
@@ -0,0 +1,62 @@
+import Chai from "chai";
+import chaiHttp from "chai-http";
+
+import app from "../../app";
+import usersTester from "./mockData/authData";
+
+Chai.should();
+Chai.use(chaiHttp);
+
+describe('Endpoint /api/v2/auth/signup', () => {
+
+    it("should create a new user account", done => {
+        Chai.request(app)
+            .post("/api/v2/auth/signup")
+            .send(usersTester[0])
+            .end((err, res) => {
+                res.should.have.status(201);
+                res.body.should.have.property("message", "User created successfully");
+                res.body.should.be.a("object");
+                done();
+            });
+    });
```

Comment:
As we have all agreed you should be using async-await in your tests. it is the Modern way of writing tests and is preferred by most developers than using callbacks.